apis/placement/v1beta1/eviction_types.go

```diff
@@ -0,0 +1,142 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package v1beta1
+
+import (
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:scope=Cluster,categories={fleet,fleet-placement},shortName=crpe
```

Comment:
do you need to print some status column using +kubebuilder:printcolumn:JSONPath=?

it can be addressed in a separate PR.